kubernetes/iac/master-compliance-test.json

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
@shahinshirvani file structure is not correct. there should be one master test name, one cases and all tests should be under cases